docs/dev-guide.md

```diff
@@ -506,10 +353,9 @@ WHERE
 - To use a JavaScript array for membership testing in SQL use [`unnest()`](https://www.postgresql.org/docs/9.5/static/functions-array.html) like:
 
   ```javascript
-  var params = {
-      id_list: [7, 12, 45],
-  };
-  sqldb.query(sql.select_questions, params, ...);
+  await sqldb.query(sql.select_questions, {
+    id_list: [7, 12, 45],
+  });
```

Comment:
I'm not sure what you have in mind here, but either use `queryAsync`, or use `queryRows` and assign to a variable.

Reply:
This was just a careless rework 🙃 fixed in 7ec3678.